docs/ai-coder/tasks.md

```diff
@@ -74,6 +86,39 @@ Because Tasks run unpredictable AI agents, often for background tasks, we recomm
 
 Alternatively, follow our guide for [custom agents](./custom-agents.md).
 
+## Agent Identity and permissions
+
+Some users may wish to or are requiredto run agents with their own identity and permissions.
+
+### Git Identity
+
+You can make use of `.gitconfig` to configure the identity of the agent. For example, you can configure the author and committer	identities separately.
+
+```tf
+resource "coder_agent" "main" {
+	...
+	env = {
+		GIT_AUTHOR_NAME = "AI Bot"
+		GIT_AUTHOR_EMAIL = "ai.bot@example.com"
+		GIT_COMMITTER_NAME = "Jane Doe"
+		GIT_COMMITTER_EMAIL = "jane.doe@example.com"
+	}
+}
+```
+
+### Permissions
+
+You have two options here to either choose the developer orchestrating the agent's permissions as you are already doing with [External Auth](https://coder.com/docs/admin/external-auth) or inject a Bot specific PAT if the tasks are started by a [headless system user](https://coder.com/docs/admin/users/headless-auth) as shown below:
+
+```tf
+resource "coder_agent" "main" {
+	...
+	env = {
+		GITHUB_TOKEN = "ghp_1234567890abcdef" # Inject a Bot specific PAT
```

Comment:
Probably best to use a managed template variable here instead of hardcoding it into the template